fontc_crater/README.md

```diff
@@ -48,6 +50,18 @@ $ cargo run --release -p=fontc_crater -- ci ../fontc_crater/gf-repos-2024-09-23.
 # Review ../fontc_crater/results/index.html (NOT ../fontc_crater/index.html)
 ```
 
+To do a full run locally, such as to test a PR (assuming same fontc_crater
+checkout as above):
+
+```shell
+# make sure the repo is up to date, so you're comparing against the latest run:
+# cd ../fontc_crater && git pull
+
+$ RUST_LOG=debug cargo run --release -p fontc_crater \
+-- ci ../fontc_crater/gf-repos-2025-02-19.json --out ../fontc_crater/results/
```

Comment:
The moving target file here makes this not reliably copy/pasteable, could we make the gf-repos file not include the date so the command is copy/pasteable?

Reply:
yea that's something I've been meaning to do anyway, will make updating much simpler.

Reply:
opened #1313  which will let us rename the input file. I'm going to merge this now, and then when that PR lands I can fix all the docs at once.